py/picca/pk1d.py

```diff
@@ -81,31 +90,49 @@ def split_forest(num_parts,
 
         log_lambda_part = log_lambda[selection].copy()
         lambda_abs_igm = constants.ABSORBER_IGM[abs_igm]
-        mean_z = (np.power(10., log_lambda_part[len(log_lambda_part) - 1]) +
-                  np.power(10., log_lambda_part[0])) / 2. / lambda_abs_igm - 1.0
+
+        if linear_binning:
+            mean_z = np.mean(log_lambda_part) / lambda_abs_igm - 1.0
```

Comment:
ftm I'm inserting either lambda (and delta_lambda) or log_lambda (and delta_log_lambda) into this function depending on binning. The function assumes linear binning in whatever data is put into it, but I agree the names need to be changed, so that it is clear that it could be either

Reply:
made function variables more verbose to make clear that either unit could be used